tests/testthat/test-create-package-docs.R

```diff
@@ -365,3 +365,83 @@ test_that("create_package_docs() changes test plan automated test section based
 
   expect_true(any(str_detect(test_plan_text, test_plan_boiler_plate)))
 })
+
+
+
+test_that("create_package_docs() works with manual tests", {
```

Comment:
As a side remark that I think is fine to not address in this series: even before this PR, there's a fair amount of duplicated test setup.  At some point, it might be worth trying to extract out some common bits so that it's easier to  see the main point of a given test.

For example, here is how this test compares to the first test in this file:

```diff
@@ -1,11 +1,11 @@
-test_that("create_package_docs() renders markdown", {
+test_that("create_package_docs() works with manual tests", {
   # set up clean docs output dir
   output_dir <- file.path(tempdir(), "mrgvalidate-create-validation-docs")
   if (fs::dir_exists(output_dir)) fs::dir_delete(output_dir)
   fs::dir_create(output_dir)
   on.exit({ fs::dir_delete(output_dir) })
 
-  specs <- readRDS(file.path(TEST_INPUTS_DIR, "specs.RDS")) %>% filter(!grepl("MAN", TestIds))
+  specs <- readRDS(file.path(TEST_INPUTS_DIR, "specs.RDS"))
   specs$StoryDescription[1] <- "story desc line 2\nLINE2!!"
 
   # Test that repeating TestIDs in traceability matrix are removed
@@ -21,6 +21,7 @@ test_that("create_package_docs() renders markdown", {
     specs = specs,
     release_notes_file = file.path(TEST_INPUTS_DIR, "release_notes_sample.md"),
     auto_test_dir = file.path(TEST_INPUTS_DIR, "validation-results-sample"),
+    man_test_dir = file.path(TEST_INPUTS_DIR, "manual-tests-sample"),
     output_dir = output_dir,
     write = TRUE,
     cleanup_rmd = FALSE
@@ -31,7 +32,8 @@ test_that("create_package_docs() renders markdown", {
 
   # get TestId's we expect to see
   test_ids <- c(
-    read_csv_test_results(file.path(TEST_INPUTS_DIR, "validation-results-sample"))$results %>% pull(TestId)
+    read_csv_test_results(file.path(TEST_INPUTS_DIR, "validation-results-sample"))$results %>% pull(TestId),
+    read_manual_test_results(file.path(TEST_INPUTS_DIR, "manual-tests-sample")) %>% pull(TestId)
   ) %>% unique()
 
 
```

Reply:
Yeah I thought about that. I was using this for my own sake as a sort of integration test to make sure everything else still behaved normally. But yeah we definitely dont need most of that. Ill redo this test